client/src/main/java/com/alibaba/nacos/client/naming/remote/NamingClientProxy.java

```diff
@@ -187,4 +188,12 @@ ListView<String> getServiceList(int pageNo, int pageSize, String groupName, Abst
      * @return true if server is healthy
      */
     boolean serverHealthy();
+
+    /**
+     * Judge nacos server whether support the ability.
+     *
+     * @param abilityKey ability key
+     * @return true if supported
+     */
+    boolean isAbilitySupportedByServer(AbilityKey abilityKey);
```

Comment:
I think the method not need define in interface. It's a gprc client proxy special method.